exercises/concept/chessboard/.docs/instructions.md

```diff
@@ -3,42 +3,42 @@
 As a chess enthusiast, you would like to write your own version of the game. Yes, there may be plenty of implementations of chess available online already, but yours will be unique!
 
 Each square of the chessboard is identified by a letter-number pair:
- - The vertical columns of squares, called files, are numbered 1 through 8.
- - The horizontal rows of squares, called ranks, are labelled A through H.
+ - The vertical columns of squares, called ranks, are numbered 1 through 8.
+ - The horizontal rows of squares, called files, are labelled A through H.
```

Comment:
```suggestion
 - The horizontal rows of squares, called ranks, are numbered 1 through 8.
 - The vertical columns of squares, called files, are labeled A through H.
```

To be correct it should be like this, right?

Reply:
Correct. Thanks for reviewing and caching that!